.github/workflows/deploy_staging.yml

```diff
@@ -0,0 +1,39 @@
+name: Deploy to Staging
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  build_and_push_image:
+    name: Build and Push Image
+    uses: zooniverse/ci-cd/.github/workflows/build_and_push_image.yaml@main
+    with:
+      repo_name: eras
+      commit_id: ${{ github.sha }}
+      latest: true
+
+  db_migration_staging:
```

Comment:
Do we need a db-migrate-staging.tmpl in order to run db migration?

Reply:
ope, I thought it was only necessary for prod but added. good catch.